src/makeConfig.js

```diff
@@ -34,13 +34,20 @@ export default function makeConfig(options) {
       ],
     },
     output: output(options),
+    optimization: {
+      minimize: true,
+      minimizer: [
+        ...optimize(options),
+      ],
+    },
     plugins: [
       ...globals(options),
       ...devtools.plugins(options),
       ...styles.plugins(options),
-      ...optimize(options),
-      ...isomorphic(options),
+      new ProgressBarPlugin(),
```

Comment:
nice plugin :) I wonder if it fits any of the other categories - maybe devtools?